discord/message.py

```diff
@@ -1991,6 +1993,9 @@ def to_message_reference_dict(self) -> MessageReferencePayload:
 
         return data
 
+    def to_dict(self) -> MessagePayload:
```

Comment:
Also, I wonder if it really makes sense to put this behind a method. This could just be `Message.data` instead of having the private attribute. See also https://github.com/Pycord-Development/pycord/issues/2498#issuecomment-2225717233